src/huggingface_hub/_shtab.py

```diff
@@ -0,0 +1,39 @@
+"""Fake shtab."""
+from argparse import Action, ArgumentParser
+
+
+FILE = None
+DIRECTORY = DIR = None
```

Comment:
What is the purpose of those global variables ?

Reply:
Because shtab provide some global variables and keep compatibility.